bom/pom.xml

```diff
@@ -130,7 +135,7 @@
       <dependency>
         <groupId>org.opendaylight.mdsal</groupId>
         <artifactId>mdsal-artifacts</artifactId>
-        <version>2.6.0-SNAPSHOT</version>
+        <version>3.0.7-SNAPSHOT</version>
```

Comment:
see #4, we could most probably actually use a fixed non-SNAPSHOT mdsal version now, just the same as on current opendaylight master?

Reply:
it's better to use `3.0.6`, because that matches the version used by controller